esm/mixin/inner-html.js

```diff
@@ -0,0 +1,19 @@
+import {parseFromString} from '../shared/parse-from-string.js';
+import {
+  ignoreCase,
+} from '../shared/utils.js';
+import {
+  CUSTOM_ELEMENTS
+} from '../shared/symbols.js';
+
+export const getInnerHtml = node => node.childNodes.join('');
```

Comment:
Please add return type via JSDoc, as the TS definition now returns `any` instead of `string`.

Reply:
Updated, now show the correct type